lib/guardian_db.ex

```diff
@@ -101,6 +111,11 @@ defmodule GuardianDb do
     end
   end
 
+  def configured_type_matches(type) do
+    env_token_type = Keyword.get(Application.get_env(:guardian_db, GuardianDb), :token_type)
```

Comment:
I think if we made it so that you could configure the token type to use a list of tokens that would be sweet